client/app/states/services/details/details.html

```diff
@@ -116,6 +130,21 @@
                     <input class="form-control" disabled value="${{ ::vm.service.chargeback.used_cost_sum | number:3 }}"/>
                   </div>
                 </div>
+                <div class="form-group">
```

Comment:
@AparnaKarve  OH apologies, was referring to everything below, the `style` stuff included in the html